File: Estrutura_Condicional/C++/Ex2344/main.cpp
#include <iostream>
using namespace std;

int main() {
    int nota;
    cin >> nota;

    if (nota == 0) {
        cout << "E" << endl;
    } else if (nota <= 35) {
        cout << "D" << endl;
    } else if (nota <= 60) {
        cout << "C" << endl;
    } else if (nota <= 85) {
        cout << "B" << endl;
    } else if (nota <= 100) {
        cout << "A" << endl;
    }

    return 0;
}

File: Estrutura_Condicional/Java/Ex2344/Main.java
package Ex2344;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int nota;
        nota = scanner.nextInt();

        if (nota == 0) {
            System.out.println("E");
        } else if (nota <= 35) {
            System.out.println("D");
        } else if (nota <= 60) {
            System.out.println("C");
        } else if (nota <= 85) {
            System.out.println("B");
        } else if (nota <= 100) {
            System.out.println("A");
        }

        scanner.close();
    }
}